framework/src/Volo.Abp.Core/Volo/Abp/Reflection/ReflectionHelper.cs

```diff
@@ -165,5 +165,42 @@ internal static void SetValueByPath(object obj, Type objectType, string property
             property = currentType.GetProperty(properties.Last());
             property.SetValue(obj, value);
         }
+
+
+        /// <summary>
+        /// Get all the constant values in the specified type (including the base type).
+        /// </summary>
+        /// <param name="type"></param>
+        /// <returns></returns>
+        public static string[] GetAllConstants(Type type)
```

Comment:
Please rename to `GetPublicConstantsRecursively`.

Reply:
sure.